genesis/Cargo.toml

```diff
@@ -20,6 +20,7 @@ name = "solana-genesis"
 path = "src/main.rs"
 
 [features]
+default = ["agave-unstable-api"]
```

Comment:
not sure why i didn't need this in #8424. must've been picking up the feature transitively from one of the crates we move here.